ansible/roles/vm_set/tasks/add_ceos_list.yml

```diff
@@ -104,6 +104,37 @@
     fp_mtu:       "{{ fp_mtu_size }}"
     max_fp_num:   "{{ max_fp_num }}"
     topo: "{{ topology }}"
+  when: not (enable_async | default(false) | bool)
+
+- name: Create VMs network (async mode)
+  become: yes
+  loop: "{{ VM_targets|flatten(levels=1) }}"
+  loop_control:
+    loop_var: vm_name
+  vm_topology:
+    cmd:          'create'
+    vm_names:     "{{ vm_name }}"
+    fp_mtu:       "{{ fp_mtu_size }}"
+    max_fp_num:   "{{ max_fp_num }}"
+    topo: "{{ topology }}"
+  async: 3600
+  poll: 0
+  throttle: 50
+  register: async_create_vm_network
+  when: enable_async | default(false) | bool
```

Comment:
How about initialize this "enable_async" variable in advance, then in the subsequent tasks, the `when` line could be simplified as either `when: enable_async` or `when: not enable_async`.

Reply:
Great idea, let me modify.

Reply:
Modified.